spec/fake_braintree/customer_spec.rb

```diff
@@ -114,6 +114,18 @@
     expect(billing_address.street_address).to eq '1 E Main St'
     expect(billing_address.postal_code).to eq '60622'
   end
+
+  it "sets the creation time" do
+    customer = Braintree::Customer.create(
+      credit_card: {
+        number: TEST_CC_NUMBER,
+        expiration_date: '04/2016'
+      }
+    ).customer
+
+    creation_time = Time.parse(customer.created_at)
```

Comment:
Do not use `Time.parse` without zone. Use one of `Time.zone.parse`, `Time.parse.current`, `Time.parse.in_time_zone`, `Time.parse.utc`, `Time.parse.getlocal`, `Time.parse.iso8601`, `Time.parse.jisx0301`, `Time.parse.rfc3339`, `Time.parse.to_i`, `Time.parse.to_f` instead.